server/protocols/webadmin-cli/src/test/java/org/apache/james/cli/MailboxManageTest.java

```diff
@@ -153,4 +153,36 @@ void mailboxExistWithNonExistingUserAndNonExistingMailboxNameShouldFail(GuiceJam
         assertThat(outputStreamCaptor.toString().trim()).isEqualTo("Either the user name or the mailbox does not exist.");
     }
 
+    @Test
+    void mailboxListWithTwoAddedMailboxesAndExistedUsernameShouldShowMailboxesNameExactly(GuiceJamesServer server) throws Exception {
+        Port port = server.getProbe(WebAdminGuiceProbe.class).getWebAdminPort();
+        dataProbe = server.getProbe(DataProbeImpl.class);
+        dataProbe.fluent().addDomain("linagora.com")
+            .addUser("hqtran@linagora.com", "123456");
+
+        WebAdminCli.executeFluent(new PrintStream(new ByteArrayOutputStream()), new PrintStream(new ByteArrayOutputStream()),
+            "--url", "http://127.0.0.1:" + port.getValue(), "mailbox", "create", "hqtran@linagora.com", "INBOX.1");
+
+        WebAdminCli.executeFluent(new PrintStream(new ByteArrayOutputStream()), new PrintStream(new ByteArrayOutputStream()),
+            "--url", "http://127.0.0.1:" + port.getValue(), "mailbox", "create", "hqtran@linagora.com", "INBOX.2");
+
+        int exitCode = WebAdminCli.executeFluent(new PrintStream(outputStreamCaptor), new PrintStream(errorStreamCaptor),
+            "--url", "http://127.0.0.1:" + port.getValue(), "mailbox", "list", "hqtran@linagora.com");
+
+        assertThat(exitCode).isEqualTo(0);
+        assertThat(outputStreamCaptor.toString()).isEqualTo("INBOX\nINBOX.1\nINBOX.2\n");
```

Comment:
That differs from your test name. Two added mailboxes but you list 3... My guess is the `.` is for sub mailboxes? So when you create `INBOX.1` you create `INBOX` and `INBOX.1` 

Maybe change your test with `Mailbox1` and `Mailbox2`. And maybe in an other test we can check we can create submailboxes without issue ?